public/js/script.js

```diff
@@ -0,0 +1,108 @@
+/*global $, document, Chart, LINECHART, data, options, window, setTimeout*/
+$(document).ready(function () {
+
+    'use strict';
+
+    // ------------------------------------------------------- //
+    // For demo purposes only
+    // ------------------------------------------------------ //
+
+    var stylesheet = $('link#theme-stylesheet');
+    $("<link id='new-stylesheet' rel='stylesheet'>").insertAfter(stylesheet);
+    var alternateColour = $('link#new-stylesheet');
+
+    if ($.cookie("theme_csspath")) {
+        alternateColour.attr("href", $.cookie("theme_csspath"));
+    }
+
+    $("#colour").change(function () {
+
+        if ($(this).val() !== '') {
+
+            var theme_csspath = 'css/style.' + $(this).val() + '.css';
+
+            alternateColour.attr("href", theme_csspath);
+
+            $.cookie("theme_csspath", theme_csspath, {
+                expires: 365,
+                path: document.URL.substr(0, document.URL.lastIndexOf('/'))
+            });
+
+        }
+
+        return false;
+    });
+
+
+    // ------------------------------------------------------- //
+    // Equalixe height
+    // ------------------------------------------------------ //
+    function equalizeHeight(x, y) {
+        var textHeight = $(x).height();
+        $(y).css('min-height', textHeight);
+    }
+    equalizeHeight('.featured-posts .text', '.featured-posts .image');
+
+    $(window).resize(function () {
+        equalizeHeight('.featured-posts .text', '.featured-posts .image');
+    });
+
+
+    // ---------------------------------------------- //
+    // Preventing URL update on navigation link click
+    // ---------------------------------------------- //
+    $('.link-scroll').bind('click', function (e) {
+        var anchor = $(this);
+        $('html, body').stop().animate({
+            scrollTop: $(anchor.attr('href')).offset().top + 2
+        }, 700);
+        e.preventDefault();
+    });
+
+
+    // ---------------------------------------------- //
+    // FancyBox
+    // ---------------------------------------------- //
+    $("[data-fancybox]").fancybox();
+
+
+    // ---------------------------------------------- //
+    // Divider Section Parallax Background
+    // ---------------------------------------------- //
+    $(window).on('scroll', function () {
+
+        var scroll = $(this).scrollTop();
+
+        if ($(window).width() > 1250) {
+            $('section.divider').css({
+                'background-position': 'left -' + scroll / 8 + 'px'
+            });
+        } else {
+            $('section.divider').css({
+                'background-position': 'center bottom'
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/mdoutreluingne/blog_php/pullRequest?prid=8164656)
```suggestion
                "background-position": 'center bottom'
```